### PR TITLE
Issue289 syncdocs

### LINKED ISSUE #289: Documentation📄: Sync Open Horizon documentation
### What is the current documentation state?

This issue will be used to sync edits between the upstream Open Horizon documentation and the downstream IBM product documentation. Some edits might flow in the both directions.

### Where is this stated?

_No response_

### Why do you want to improve the statement?

_No response_

### Proposed Statement

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by johnwalicki**
# Pull Request Template

## Description

This PR syncs some of the Open Horizon and IBM product documentation so they are in sync.
- Lint fixes, etc. 

Fixes #289 

## Type of change

- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

Visit the following pages after the PR has been merged:

- https://open-horizon.github.io/docs/hub/cluster_sizing.html
- https://open-horizon.github.io/docs/developing/further_reading_devices.html
- https://open-horizon.github.io/docs/developing/service_containers.html
- https://open-horizon.github.io/docs/getting_started/faq.html
- https://open-horizon.github.io/docs/getting_started/components.html
- https://open-horizon.github.io/docs/hub/configuration.html
- https://open-horizon.github.io/docs/installing/advanced_man_install.html
- https://open-horizon.github.io/docs/installing/edge_cluster_agent.html
- https://open-horizon.github.io/docs/using_edge_services/cpu_load_example.html
- https://open-horizon.github.io/docs/installing/sdo.html
...

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by johnwalicki** [comment]
I added more commits based on page rendering reviews

**Comment by johnwalicki** [comment]
@joewxboy  Can you do a review of this PR?   Would be nice to have this as an accomplishment for the OH Docs WG call tomorrow (Tues 7/12)

File: docs/getting_started/faq.md [hunk 1]
```diff
@@ -36,31 +35,26 @@ The following are answers to some frequently asked questions (FAQs) about {{site
...
 {: #open_sourced}
 
-{{site.data.keyword.ieam}} is an IBM product. But its core components heavily use the [Open Horizon - EdgeX Project Group ](https://wiki.edgexfoundry.org/display/FA/Open+Horizon+-+EdgeX+Project+Group){:target="\_blank"}{: .externalLink} open source project. Many sample and example programs that are available in the {{site.data.keyword.horizon}} project work with {{site.data.keyword.ieam}}. For more information about the project, see [Open Horizon - EdgeX Project Group ](https://wiki.edgexfoundry.org/display/FA/Open+Horizon+-+EdgeX+Project+Group){:target="\_blank"}{: .externalLink}.
+{{site.data.keyword.ieam}} is an IBM product. But its core components heavily use the [Open Horizon - EdgeX Project Group ](https://wiki.edgexfoundry.org/display/FA/Open+Horizon+-+EdgeX+Project+Group){:target="_blank"}{: .externalLink} open source project. Many sample and example programs that are available in the {{site.data.keyword.horizon}} project work with {{site.data.keyword.ieam}}. For more information about the project, see [Open Horizon - EdgeX Project Group ](https://wiki.edgexfoundry.org/display/FA/Open+Horizon+-+EdgeX+Project+Group){:target="_blank"}{: .externalLink}.
```
Comment (joewxboy) [inline docs/getting_started/faq.md:45]: The following link is legacy from before Open Horizon was a standalone project:

[Open Horizon - EdgeX Project Group ](https://wiki.edgexfoundry.org/display/FA/Open+Horizon+-+EdgeX+Project+Group)

Suggest changing it to the official project page at:

[Open Horizon, an LF Edge project](https://www.lfedge.org/projects/openhorizon/)

**Review by joewxboy — APPROVED** [review]
There's one area where I left comments about a legacy link to information about Open Horizon that can be updated.

File: docs/getting_started/faq.md (hunk 1 above)
Comment (johnwalicki) [reply docs/getting_started/faq.md]: I fixed the LFEdge Open Horizon URL in the FAQ in https://github.com/open-horizon/open-horizon.github.io/pull/290/commits/193d1a850b5021a46b6a1ac6a0816af0886fa9e8

Comment (johnwalicki) [reply docs/getting_started/faq.md]: Thanks for the review.

**Comment by johnwalicki** [comment]
See published page review and new issues at https://github.com/open-horizon/open-horizon.github.io/issues/289#issuecomment-1181234990

**Pull request merged**